src/Value/Color/HexColor.php

```diff
@@ -0,0 +1,48 @@
+<?php
+declare(strict_types=1);
+
+namespace MyOnlineStore\Common\Domain\Value\Color;
+
+use MyOnlineStore\Common\Domain\Exception\Color\InvalidHexColor;
+
+/**
+ * @final
```

Comment:
since we know we need to extend the VO in the `store-branding-service` because of the Doctrine Embedable we use there. Let's drop the @final and make the value protected (so it can be annotated)
```suggestion
```

Reply:
It's up to you, but I would advise to not use an embeddable, but a custom doctrine type instead,
- It is no longer needed to add doctrine mapping to your VO (thus this VO could also be final)
- You can have nullable values (embeddables can't be null)
- You'll have a much cleaner API when writing queries

See for example the tax-policy service, where we used this

Reply:
I really hate embedabbles 😆 so let's go with Franks proposal